source/security/enterprise-authentication.txt

```diff
@@ -0,0 +1,410 @@
+
+.. tabs::
+
+   .. tab::
+      :tabid: Connection String
+
+      .. literalinclude:: /includes/security/enterprise-authentication.java
+         :start-after: // start-ldap-connection-string
+         :end-before: // end-ldap-connection-string
+         :language: java
+
+   .. tab::
+      :tabid: MongoCredential
+
+      To specify the LDAP (PLAIN) authentication mechanism by using the
+      ``MongoCredential`` class, call the ``createPlainCredential()``
+      method, as shown in the following example:
+
+      .. literalinclude:: /includes/security/enterprise-authentication.java
+         :start-after: // start-ldap-mongocredential
+         :end-before: // end-ldap-mongocredential
+         :language: java
+
+.. _java-rs-mongodb-oidc:
+
+MONGODB-OIDC
+~~~~~~~~~~~~
+
+.. important::
+
+   The MONGODB-OIDC authentication mechanism requires {+mdb-server+} v7.0 or later running
+   on a Linux platform.
+
+The following sections describe how to use the MONGODB-OIDC authentication mechanism to
+authenticate to various platforms.
+
+For more information about the MONGODB-OIDC authentication mechanism, see
+:manual:`OpenID Connect Authentication </core/security-oidc/>` and
+:manual:`MongoDB Server Parameters </reference/parameters/#mongodb-parameter-param.oidcIdentityProviders>`
+in the MongoDB Server manual.
+
+.. _java-rs-mongodb-oidc-azure-imds:
+
+Azure IMDS
+++++++++++
+
+If your application runs on an Azure VM, or otherwise uses the
+`Azure Instance Metadata Service <https://learn.microsoft.com/en-us/azure/virtual-machines/instance-metadata-service>`__
+(IMDS), you can authenticate to MongoDB by using the {+driver-short+}'s built-in Azure
+support.
+
+You can specify Azure IMDS OIDC authentication either by 
+using a ``MongoCredential`` or as part of the connection string.
+
+Select the :guilabel:`Connection String` or :guilabel:`MongoCredential` tabs to
+see the corresponding syntax.
+
+.. tabs::
+
+   .. tab:: Connection String
+      :tabid: mongodb-azure-imds-connection-string
+    
+      Replace the ``<percent-encoded audience>`` placeholder in the
+      following code with the percent-encoded value of the audience server
+      parameter configured on your MongoDB deployment.
+
+      The comma (``,``) character and its encoding (``%2C``) are
+      reserved, and using these characters in a value causes the
+      driver to interpret commas as delimiters of key-value pairs.
+      You must specify values that contain commas in a ``MongoCredential`` instance, as
+      demonstrated in the :guilabel:`MongoCredential` tab.
+     
+      .. literalinclude:: /includes/security/enterprise-authentication.java
+         :start-after: // start-azure-oidc-connection-string
+         :end-before: // end-azure-oidc-connection-string
+         :language: java
+
+   .. tab:: MongoCredential
+      :tabid: mongodb-azure-mongo-credential
+
+      Replace the ``<username>`` placeholder with the client ID or application ID of the
```

Comment:
S:
```suggestion
      Replace the ``<db_username>`` placeholder with the client ID or application ID of the
```

Reply:
Leaving this now that all are changed to `username`